crates/astria-sequencer/src/app.rs

```diff
@@ -442,133 +507,103 @@ impl App {
             );
         }
 
+        self.execution_results = Some(execution_results);
         Ok((signed_txs, validated_txs))
```

Comment:
nit: feels kinda overloaded here with the mutation of property on self and then also returning a tuple. I can't think of a great fix right now, but it feels a tad off.

Reply:
yeah it's more like this (returning the tuple) to prevent unnecessary allocations